File: 1012.cpp
#include <stdio.h>
 
int main() {
 
   double a,b,c;
   
   scanf("%lf %lf %lf",&a,&b,&c);
   
   printf("TRIANGULO: %.3lf\n", a * c /2);
   
   printf("CIRCULO: %.3lf\n",3.14159 * c * c);

	printf("TRAPEZIO: %.3lf\n", c * (a + b) /2);
	
	printf("QUADRADO: %.3lf\n", b * b);
	
	printf("RETANGULO: %.3lf\n", a * b);  
   
}

//Vinicius Mangueira Correia

File: Answers/Python/1012.py
# -*- coding: utf-8 -*-

pi = 3.14159
A, B, C = map(float, raw_input().split())

triangulo = (A*C)/2

circulo = (C*C)*pi

trapezio = ((A+B)*C)/2

quadrado = B*B

retangulo = A*B

print("TRIANGULO: %.3f" %triangulo)
print("CIRCULO: %.3f" %circulo)
print("TRAPEZIO: %.3f" %trapezio)
print("QUADRADO: %.3f" %quadrado)
print("RETANGULO: %.3f" %retangulo)